thrust/system/cuda/detail/synchronize.h

```diff
@@ -32,11 +32,15 @@ namespace detail
 inline __host__ __device__
 void synchronize(const char *message = "");
 
+inline __host__ __device__
+void synchronize(cudaStream_t, const char *message = "");
```

Comment:
May as well give this parameter a name. Without one, it gives the impression that the parameter is unused.